core/utils/histogram.h

```diff
@@ -131,9 +134,12 @@ class Histogram {
     return ret;
   }
 
+  // Resets all counters, and the count of such counters, to zero.
+  // Note that the number of buckets remains unchanged.  We assume
+  // memset-to-zero works for all arithmetic types.
   void Reset() {
     count_ = 0;
-    buckets_ = std::vector<size_t>(buckets_.size());
+    memset(buckets_.data(), 0, buckets_.size() * sizeof(T));
```

Comment:
Why not the more C++ style approach?

`std::fill(buckets_.begin(), buckets_.end(), 0);`

My understanding is that it will be optimized to be roughly the same performance.

Reply:
If it does optimize properly, that would be better.  I'll check it out.  It's not very important since bess doesn't use the Reset() function anyway though...